guides/common/modules/proc_enabling-the-flatpak-remote.adoc

```diff
@@ -0,0 +1,71 @@
+[id="enabling-the-flatpak-remote"]
+= Enabling the Flatpak remote
+
+This Hammer CLI procedure enables and manages Flatpak repositories.
+
+.Prerequisites
+* *view_flatpak_remotes*: The user can view Flatpak remotes.
+* *create_flatpak_remotes*: The user can create new Flatpak remotes.
+* *edit_flatpak_remotes*: Grants the user permission to change existing Flatpak remotes.
+* *destroy_flatpak_remotes*: The user can delete Flatpak remotes.
+
+Set up Flatpak. 
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/assembly_installing-applications-using-flatpak_using-the-desktop-environment-in-rhel-8#setting-up-flatpak_assembly_installing-applications-using-flatpak[Setting up Flatpak].
+
+Ensure the assigned user role includes these permissions to manage Flatpak remotes without admin privileges.
+
+Flatpak applications rely on Flatpak runtimes. 
+For example, `rhel9/firefox-flatpak` depends on `rhel9/flatpak-runtime`. 
+Ensure that runtime repositories are available to clients alongside application repositories for installations to work.
+
+.CLI procedure
+. Enable a Flatpak remote on the {ProjectServer} by using standalone {EL} systems or {ProjectName}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ hammer flatpak-remote create --name=_My_Name_ --url=$url \
+--organization=your_organization
+----
++
+You can include authentication details during creation by using the options `--username=$username --token=$token`.
++
+You can generate a {Team} official token at link:https://access.redhat.com/terms-based-registry[Registry Service Accounts].
+. Update the Flatpak remote with authentication credentials:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ hammer flatpak-remote update --username=_My_User_Name_ --token=_My_Token_
+----
+. List and view information about the Flatpak remote:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ hammer flatpak-remote list
+$ hammer flatpak-remote info
+----
```

Comment:
This is not really part of 'enabling Flatpak remote'. You can either drop the whole step (users will be able to find this in hammer --help or in the Hammer guide or the release notes, actually), or you can at least mark it as an optional step.

One reason why I'm suggesting this is that it's good to keep in mind that we don't want our procedures to be too long. Long procedures are more difficult to read and can even look intimidating at first glance. If you're writing a procedure on 'enabling a Flatpak remote', then it's good to include steps relevant to that particular user goal rather than try to document all the commands that have been added.